### DISCUSSION TITLE
Is there a good remote desktop for hyprland?

### DISCUSSION THREAD

**Discussion by Gedankenn** [prasanthrangan/hyprdots#104] [Q&A]
Im trying to get remote acess for my linux and i'm having some dificulties, anyone has an insight that may help?
Thanks in advance.

**Comment by T-Crypt** [comment]
You may want to look at SPICE. RDP is a little more difficult on Hyprland.

**Reply by Gedankenn** [reply]
I looked in it, got some trouble in understand how to create a spice server so i can connect, the way i found is using x11spice. if you could elaborate a little more would be really helpfull. 

Really thanks for the reply, it was exactly what i wanted, i just couldnt get my head around making it work.
